driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -304,15 +315,45 @@ public long fetchSize() {
 
     /**
      * Returns notification config.
+     *
      * @return the notification config
      * @since 5.7
+     * @deprecated superseded by {@link #minimumNotificationSeverity()} and
+     * {@link #disabledNotificationClassifications()}
      */
+    @Deprecated
     public NotificationConfig notificationConfig() {
         return notificationConfig;
     }
 
     /**
-     * Returns the number of {@link io.netty.channel.EventLoop} threads.
+     * Returns a minimum notification severity.
+     *
+     * @return an {@link Optional} of minimum {@link NotificationSeverity} or an empty {@link Optional} if it is not set
+     * @since 5.22.0
+     */
+    public Optional<NotificationSeverity> minimumNotificationSeverity() {
+        return Optional.ofNullable(((InternalNotificationConfig) notificationConfig).minimumSeverity());
+    }
+
+    /**
+     * Returns a set of disabled notification classifications.
+     * @return the {@link Set} of disabled {@link NotificationClassification}
+     * @since 5.22.0
+     */
+    public Set<NotificationClassification> disabledNotificationClassifications() {
```

Comment:
Methods and new types introduced should be marked as preview, we wan't to let our interface decisions open for while.

Reply:
Good point, done. 👍